packages/sdk/src/react/hooks.ts

```
@@ -89,205 +95,178 @@ export function useEntityId(...keys: BigNumberish[]): BigNumberish {
-            await sleep(50);
-        }
+function createSubscriptionHook<
+    Schema extends SchemaType,
+    Historical extends boolean = false,
+>(config: {
+    subscribeMethod: (
+        options: SubscribeParams<Schema, Historical>
+    ) => Promise<SubscribeResponse<Schema, Historical>>;
+    updateSubscriptionMethod: (
+        subscription: Subscription,
+        clause: any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)